backend/prisma/migrations/20240729210931_init_favourites_table/migration.sql

```diff
@@ -0,0 +1,21 @@
+-- CreateTable
+CREATE TABLE "Favourite" (
+    "id" SERIAL NOT NULL,
+    "UserId" INTEGER NOT NULL,
+    "adopterId" INTEGER,
```

Comment:
If the user is an adoptee and likes the profile of an adopter the adopterId is stored. Therefore the adopter id is the number of the adopter being liked.